pkg/data/utils.go

```diff
@@ -33,16 +33,48 @@ func GetJobOfferID(offer JobOffer) (string, error) {
 	return CalculateCID(offer)
 }
 
+func GetJobOfferContainerIDs(jobOffers []JobOfferContainer) []string {
+	var ids []string
+	for _, offer := range jobOffers {
+		ids = append(ids, offer.ID)
+	}
+	return ids
+}
+
 func GetResourceOfferID(offer ResourceOffer) (string, error) {
 	offer.ID = ""
 	return CalculateCID(offer)
 }
 
+func GetResourceOfferIDs(resourceOffers []ResourceOffer) []string {
+	var ids []string
+	for _, offer := range resourceOffers {
+		ids = append(ids, offer.ID)
+	}
+	return ids
+}
```

Comment:
It may be confusing that that `GetResourceOfferIDs` has a different behavior than `GetResourceOfferID`. Open to naming suggestions.